data_override/data_override.F90

```diff
@@ -115,6 +115,29 @@ module data_override_mod
    integer            :: region_type
 end type data_type
 
+type nudge3D
+   character(len=3)                 :: gridname
+   character(len=128)               :: varname
+   character(len=128)               :: obs_coefficient_name
+   character(len=128)               :: mod_coefficient_name
+   real, _ALLOCATABLE               :: model_input(:,:,:) _NULL
+   real, _ALLOCATABLE               :: nudge_output(:,:,:) _NULL
+   real, _ALLOCATABLE               :: observed_data(:,:,:) _NULL
+   real, _ALLOCATABLE               :: observation_coefficient(:,:,:) _NULL
+   real, _ALLOCATABLE               :: model_coefficient(:,:,:) _NULL
```

Comment:
IMHO, these `_ALLOCATABLE` and `_NULL` should just be allocatable and no null is needed
```fortran
    real, ALLOCATABLE               :: model_input(:,:,:) 
￼   real, ALLOCATABLE               :: nudge_output(:,:,:) 
￼   real, ALLOCATABLE               :: observed_data(:,:,:) 
￼   real, ALLOCATABLE               :: observation_coefficient(:,:,:) 
￼   real, ALLOCATABLE               :: model_coefficient(:,:,:) 
```

Reply:
changed as suggested